ironfish-cli/src/commands/deposit-all.ts

```diff
@@ -111,7 +121,7 @@ export default class DepositAll extends IronfishCommand {
       this.client,
       this.api,
       expirationSequenceDelta,
-      fee,
+      Number(fee),
```

Comment:
This is still broken, and all of this needs to be fixed in another PR. verifyCanSend is broken because it's accepting ore but treating it like iron.